File: client/src/app/shared/models/consulta-result-model.ts
export interface ConsultaResultModel {
    dataHora: Date;
    estacao: number;
    tempAr: number;  
    tempMin: number; 
    tempMax: number;
    tempOrv: number;
    chuva: number;
    direcaoVento: number;
    velocidadeVento: number;
    velocidadeVentoMax: number;
    bateria: number;
    radiacao: number;
    pressaoATM: number;
    indiceCalor: number;
    umidadeRelativa: number;
}

File: server/MeteoroCefet.Application/Models/ConsultaResultModel.cs

namespace MeteoroCefet.Application.Models
{
    public class ConsultaResultModel
    {
        public DateTime DataHora { get; set; }
        public double Estacao { get; set; }
        public double TempAr { get; set; }
        public double TempMin { get; set; }
        public double TempMax { get; set; }
        public double TempOrv { get; set; }
        public double Chuva { get; set; }
        public double DirecaoVento { get; set; } 
        public double VelocidadeVento { get; set; }
        public double VelocidadeVentoMax { get; set; }
        public double Bateria { get; set; }
        public double Radiacao { get; set; }
        public double PressaoATM { get; set; }
        public double IndiceCalor { get; set; }
        public double UmidadeRelativa { get; set; }
    }
}
